lib/internal/Magento/Framework/App/FeedFactoryInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedFactoryInterface
+{
+
+    const FORMAT_RSS = 'rss';
+
+    /**
+     * Returns FeedInterface object from a custom array
+     * 
+     * @throws \Magento\Framework\Exception\InputException
```

Comment:
`throws \Magento\Framework\Exception\RuntimeException`
should be added to the contract